client/src/services/return-of-service.js

```diff
@@ -61,3 +61,33 @@ export const getAllSites = async () => {
     throw new Error('Failed to fetch all sites');
   }
 };
+
+export const updateRosStatus = async (participantId, newValues) => {
+  const url = `${API_URL}/api/v1/ros/participant/${participantId}`;
+  const { siteName, startDate, date } = newValues;
+
+  if (!siteName && !startDate && !date) {
+    throw new Error('Unable  to update the field - no changes found');
+  }
+  if (!participantId) {
+    throw new Error('Unable  to update the field - invalid data');
+  }
+  const dateTimestamp = date ? dayjs(date, 'YYYY/MM/DD').toDate() : undefined;
+  const startDateTimestamp = startDate ? dayjs(startDate, 'YYYY/MM/DD').toDate() : undefined;
+
+  return fetch(url, {
+    method: 'PATCH',
```

Comment:
Are we back to where we started here? In this PR, are we:
- Sending a PATCH request from the frontend to update ROS status
- In the PATCH endpoint, we are creating a new ROS status for this participant and invalidating any previous ones, so as to create an audit log

Really sorry, am I misunderstanding what's happening?